src/helpers.js

```diff
@@ -9,7 +9,9 @@ firebase.initializeApp(firebaseConfig);
 const db = firebase.firestore();
 const storage = firebase.storage();
 const collectionRef = db.collection("carts");
-const getQuery = db.collectionGroup("carts").orderBy("timestamp", "asc");
+const getCartsQuery = db.collectionGroup("carts").orderBy("timestamp", "asc");
+const getConstantsQuery = db.doc("/triggers/constants");
```

Comment:
Constants are constants like the duration the leave animation should take, how long before the train leaves should we display the timer, etc. These constants are stored in firestore because the cloud function uses them too